doc/2/plugins/plugin-context/errors/partialerror/index.md

```diff
@@ -21,7 +21,7 @@ new context.error.PartialError(message, failures);
 | Arguments  | Type                                                               | Description                |
 | ---------- | ------------------------------------------------------------------ | -------------------------- |
 | `message`  | <pre>string</pre>                                                  | Error message              |
-| `failures` | [`KuzzleError[]`](/core/1/plugins/errors/kuzzleerror) | List of encountered errors |
+| `failures` | [`KuzzleError[]`](/core/2/plugins/errors/kuzzleerror) | List of encountered errors |
```

Comment:
``[`KuzzleError[]`](/core/2/plugins/errors/kuzzleerror)`` lead to 404 and link hard to see

![image](https://user-images.githubusercontent.com/19224148/63167144-afafad80-c030-11e9-90b3-ea7ded935b6a.png)